submissions/OlStaniev/OOP exercise/index.js

```diff
@@ -0,0 +1,43 @@
+class Inhabitant {
+   constructor(specie, name, gender, legs, saying) {
+      this.specie = specie
+      this.name = name
+      this.gender = gender
+      this.legs = legs
+      this.saying = saying
+   }
+   prepareToPrint() {
+      return [this.specie, this.name, this.gender, this.legs, this.saying].join('; ')
+   }
+}
+
+class Person extends Inhabitant {
+   constructor(specie, name, gender, legs, hands, saying, friends) {
+      super(specie, name, gender, legs, saying)
```

Comment:
We assume all instances of `Person` are of human species.
We may let users skip passing this property and send a scalar string/constant to the base class' constructor.